docs/Spring Boot/5_Step_To_Make_Gradle_Configuration_Extreme_Clean_In_A_Multi-Module_Project.md

```diff
@@ -0,0 +1,216 @@
+# 5 Step To Make Gradle Configuration Extreme Clean In a Multi-Module Project
+
+In a multi-module Gradle project, there are many tasks to handle during the build process. Using Gradle to manage dependency version control, plugin usage, build logic, and more is a popular and effective approach. However, achieving these tasks requires a lot of configuration scripts, which can make the file more complicated, and more difficult for development. In this article, we will demonstrate a clean and efficient way to manage configuration files. The needed steps are as follows:
```

Comment:
⚠️ **[vale]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[Google.We] Try to avoid using first-person plural like 'we'.